src/keri/app/cli/commands/escrow/list.py

```diff
@@ -0,0 +1,279 @@
+
+            if (not escrow) or escrow == "reply":
+                count = 0
+                for key, _ in hby.db.rpes.getItemIter():
+                    count += 1
+                escrows["reply"] = count
+
+            if (not escrow) or escrow == "failed-oobi":
+                count = 0
+                for key, _ in hby.db.eoobi.getItemIter():
+                    count += 1
+                escrows["failed-oobi"] = count
+
+            if (not escrow) or escrow == 'group-partial-witness':
+                count = 0
+                for key, _ in hby.db.gpwe.getItemIter():
+                    count += 1
+                escrows["group-partial-witness"] = count
+
+            if (not escrow) or escrow == 'group-delegate':
+                count = 0
+                for key, _ in hby.db.gdee.getItemIter():
+                    count += 1
+                escrows["group-delegate"] = count
+
+            if (not escrow) or escrow == 'delegated-partial-witness':
+                count = 0
+                for key, _ in hby.db.dpwe.getItemIter():
+                    count += 1
+                escrows["delegated-partial-witness"] = count
+
+            if (not escrow) or escrow == 'group-partial-signed':
+                count = 0
+                for key, _ in hby.db.gpse.getItemIter():
+                    count += 1
+                escrows["group-partial-signed"] = count
+
+            if (not escrow) or escrow == 'exchange-partial-signed':
+                count = 0
+                for key, _ in hby.db.epse.getItemIter():
+                    count += 1
+                escrows["exchange-partial-signed"] = count
+
+            if (not escrow) or escrow == 'delegated-unanchored':
+                count = 0
+                for key, _ in hby.db.dune.getItemIter():
+                    count += 1
+                escrows["delegated-unanchored"] = count
+
+            # TEL / Reger escrows
+            reger = viring.Reger(name=hby.name, db=hby.db, temp=False)
+
+            if (not escrow) or escrow == 'tel-out-of-order':
+                count = 0
+                for key, _ in reger.getOotItemIter():
+                    count += 1
+                escrows["tel-out-of-order"] = count
+
+            if (not escrow) or escrow == 'tel-partially-witnessed':
+                count = 0
+                for key, _ in reger.getAllItemIter(reger.twes):
+                    count += 1
+                escrows["tel-partially-witnessed"] = count
+
+            if (not escrow) or escrow == 'tel-anchorless':
+                count = 0
+                for key, _ in reger.getAllItemIter(reger.taes):
+                    count += 1
+                escrows["tel-anchorless"] = count
+
+            if (not escrow) or escrow == "missing-registry-escrow":
+                creds = list()
+                for (said,), dater in reger.mre.getItemIter():
+                    creder, *_ = reger.cloneCred(said)
```

Comment:
Why `*_` here?

Reply:
Likely brevity as there are four return arguments in the tuple and only the first was needed.. Someone else wrote this code.